vlib/v/checker/tests/invalid_insert_references_test.out

```diff
@@ -0,0 +1,7 @@
+vlib/v/checker/tests/invalid_insert_references_test.vv:8:7: error: cannot append `int literal` to `[]&int`. you can prefix the right argument with &.
```

Comment:
I would remove the "you can prefix the ..."

you can't do `&1`  for example

the first part of the error is enough

Reply:
A little confusing, existing programs do exist similar &1.

Reply:
ok: we *can* detect those cases to give a precise hint probably, but it might be simpler to skip it for now